warehouse/warehouse/scripts/fix_time_units.py

```diff
@@ -11,8 +11,9 @@ def fix_units(inpath, outpath, time_units, offset):
     import xarray as xr
     with xr.open_dataset(inpath, decode_times=False) as ds:
         if ds.get('time') is None:
-            con_message('error',f"{inpath} has no 'time' axis")
-            raise ValueError(f"{inpath} has no 'time' axis")
+            con_message('error',f"{os.path.basename(inpath)} has no 'time' axis")
+            exit(1)
+            # raise ValueError(f"{inpath} has no 'time' axis")
```

Comment:
remove the comment, the previous version is recorded in git if we need to reference it

Reply:
ok, sure.